vividus/src/test/java/org/vividus/expression/StringsExpressionProcessorTests.java

```diff
@@ -85,7 +85,9 @@ static Stream<Arguments> expressionSource()
                 arguments("toBase64Gzip(vividus)",                                         "H4sIAAAAAAAA/yvLLMtMKS0GANIHCdkHAAAA"),
                 arguments("escapeHTML(M&Ms)",                                              "M&amp;Ms"),
                 arguments("escapeJSON(\"abc\"\n\"xyz\")",                                  "\\\"abc\\\"\\n\\\"xyz\\\""),
-                arguments("quoteRegExp(Customer(Username))",                               "\\QCustomer(Username)\\E")
+                arguments("quoteRegExp(Customer(Username))",                               "\\QCustomer(Username)\\E"),
+                arguments("substringBefore(123, 2)",                                       "1"),
+                arguments("substringAfter(123, 2)",                                        "3")
```

Comment:
```suggestion
                arguments("substringAfter(123, 2)",                                        "3"),
                arguments("substringAfter(Hello\\, boddissattva, Hello\\,)",                "boddissattva")
```

I believe these expressions should also cover similar cases

Reply:
Thanks for your review.
I added another test to handle separator's spaces.
The generic method **substring** could be used by any sort of bi string functions. Therefore, do you prefer another name ?

Reply:
@boddissattva can you please check failures in CI and fix them?

Reply:
Should be ok now.